permissions/plugin-alibabacloud-credentials-plugin.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "alibabacloud-credentials"
+github: "jenkinsci/alibabacloud-credentials-plugin"
+paths:
+- "org/jenkins-ci/plugins/alibabacloud-credentials"
```

Comment:
wrong group id:

https://github.com/jenkinsci/alibabacloud-credentials-plugin/blob/master/pom.xml#L10

Reply:
This question has been revised